### DISCUSSION TITLE
Alternatives to loop functionality.

### DISCUSSION THREAD

**Discussion by qkxie** [langgenius/dify#4290] [Suggestion]
Since Dify currently doesn't support loop functionality, we need to achieve looping indirectly through its API. For instance, your original expected workflow was:

Input -> LLM -> A list -> Perform the subsequent process for each item in the list -> Latter...

Now, using Dify's API functionality, we can split the workflow into two workflows.

The first workflow:

Input -> LLM -> A list

Then, we use a Python for loop to pass the contents of the array one by one into the second workflow.

**Comment by patryk20120** [comment]
If it is a Feature Request, then as an addition - Dify team is working on Loop functionality.
